dashboard/app/views/levels/editors/_artist.html.haml

```diff
@@ -22,6 +22,12 @@
 .field
   = f.label :permitted_errors, 'Permitted errors (default 0)'
   = f.number_field :permitted_errors, :value => @level.permitted_errors
+.field
+  = f.label :auto_run, 'Automatically rerun on block changes'
+  = boolean_check_box f, :auto_run
+.field
+  = f.label :limited_auto_run, 'Automatically rerun on block changes, but only allow field edits'
+  = boolean_check_box f, :limited_auto_run
```

Comment:
should this instead be two potential modes for the single `auto_run` field?

Reply:
Yup, done.